docs/user/interactive_basics.rst

```diff
@@ -15,40 +15,39 @@ of code before combining everything into a (reproducible) workflow.
 You can run JupyterLab or RStudio within a project independently from RenkuLab,
 but RenkuLab offers the following advantages:
 
-* environments hosted in the cloud with a configurable amount of resources
-  (memory, CPU, and sometimes GPU)
+* Environments hosted in the cloud with a configurable amount of resources
+  (memory, CPU, and sometimes GPU).
 
-* environments are defined using Docker, so they can be shared and reproducibly
-  re-created
+* Environments are defined using Docker, so they can be shared and reproducibly re-created.
 
-* auto-saving of work back to RenkuLab, so you can recover in the event of a
-  crash
+* Auto-saving of work back to RenkuLab, so you can recover when your environment is shut down
+  (this happens automatically after 24 hours of inactivity).
 
-* a git client pre-configured with your credentials to easily push your changes
-  back to the server
+* A git client pre-configured with your credentials to easily push your changes
+  back to the server.
 
-* the functionality provided by the renku-python_ command-line interface (CLI)
-  is automatically available
+* The functionality provided by the renku-python_ command-line interface (CLI)
+  is automatically available.
 
 
 What's in my Interactive Environment?
 -------------------------------------
 
-* your project, which is cloned into the environment on startup
+* Your project, which is cloned into the environment on startup.
 
-* your data (if the option ``Automatically fetch LFS data`` is selected)
-  files that are stored in git LFS*)
+* Your data files (if the option ``Automatically fetch LFS data`` is selected)
+  that are stored in git LFS*.
 
-* all the software required to launch the environment and common tools for
-  working with code (``git``, ``git LFS``, ``vim``, etc.)
+* All the software required to launch the environment and common tools for
+  working with code (``git``, ``git LFS``, ``vim``, etc.).
 
-* any dependencies you specified via conda (requirements.txt), using
+* Any dependencies you specified via conda (``environment.yml``), using
   language-specific dependency-management facilities (``requirements.txt``,
-  ``install.R``, etc.) or installed in the ``Dockerfile``
+  ``install.R``, etc.) or installed in the ``Dockerfile``.
```

Comment:
We should add to this, I think.

```
* Any dependencies you specified via conda (``environment.yml``), using language-specific dependency-management facilities (``requirements.txt``, ``install.R``, etc.) or installed in the ``Dockerfile``. (An exception to this is if the project uses a pinned image. [See the section below](linked to pinned image section) for more information.)
```

Reply:
@ciyer I thought we should try to avoid using the word "pinned" - I don't think it's really a term that most of our users understand immediately. But maybe it's more widely used than I think?

Reply:
@rokroskar I wasn't involved with those discussions, so I don't know what arguments were presented. 

I just do not find "custom" very descriptive in this case. There image isn't any more or less customized than any other image used in an interactive environment -- what is unique in this case is that the image if pinned/fixed and may not reflect the contents of the project (Dockerfile, environment.yml, etc.).

In fact, the project that was created for testing was called "fixed-image" -- that's an indication to me that there might be a naming mismatch.

Reply:
@ciyer there wasn't really a discussion, it was just a suggestion I made [here](https://github.com/SwissDataScienceCenter/renku/pull/1737#discussion_r530839218) - I'm fine with using "pinned" but I thought we could maybe come up with something a bit less technical. If there isn't a better term then pinned is also OK. 

It's custom in the sense that it's not the one automatically provided by the system. It might be one of our standard images, one that was built by a previous CI run, or something that the user created completely on their own.